__mocks__/axios.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * @file axios.js
+ * Exports an Axios mock.
+ */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+import mockAxios from 'jest-mock-axios';
+
+export default {
```

Comment:
This is my axios mock. It relies on `jest-mock-axios` to do most of the work, but adds support for interceptors and the axios `create` method.